Bring back "Back to Dashboard link"

mrchief opened the pull request:
Not sure if this was intentional but it looked kind of odd to me. 

**before:**

![image](https://cloud.githubusercontent.com/assets/781818/12615683/1fbe8c50-c4d5-11e5-9653-6878f099f12a.png)

---

**after:**

![image](https://cloud.githubusercontent.com/assets/781818/12615684/25eb2110-c4d5-11e5-8bc5-d2d022ade930.png)

afonsof commented:
What's your browser and jenkins version?

It should look like this:
![image](https://cloud.githubusercontent.com/assets/805511/12616147/bea7a386-c4f0-11e5-8076-3309f0d95a51.png)

Have you made changes only in the color scheme?

mrchief commented:
Ahh... I see now. I was trying to bring back the breadcrumb bar and made changes to the header like so (to make the breadcrumb bar stack after the header):

```
#page-head {
  position: fixed;
  right: 0;
  left: 0;
  top: 0;
  z-index: 3;
  ...
}

#header {
  ...
  position: relative;
}
```

That would have messed it. 

Yeah, I changed to the color to match the client's branding needs. I also figured a way to replace the Jenkins logo with the client's. Let me know if you are interested in the logo change. Will be happy to send a PR or just comment in here.

Testing on Chrome 47.0.2526.111 m, Jenkins 1.613

afonsof commented:
Ok, that is not a bug and the breadcrumbs problem is fixed, so I'm closing this one :smile:

mrchief commented:
How does it look with the Jenkins logo in place? Can you send a larger screenshot?

afonsof commented:
In that case Jenkins logo disappears and the arrow takes it's place. I did this in order to work like a default Material Design app.
![image](https://cloud.githubusercontent.com/assets/805511/12619252/605a7c78-c4fe-11e5-84ca-2e226409f46c.png)

mrchief commented:
Oh I see. I'm split now... A part of me wants to embrace this but then, another part of me wants the logo visible all the time, but that would probably look "not so good" next to the back icon :|

afonsof commented:
Yup, I know what you mean... I've tried both together and they were not ok :disappointed:

mrchief commented:
Let me see if I can find a good compromise.

Unrelated: Did you try running and connecting to the vagrant box? I can ssh to it but cannot connect to Jenkins on it. `curl 'http://localhost:8080'` on the box fails and not much connecting from outside `http://10.0.2.15:8880` & `http://localhost:8880` fails as well.

afonsof commented:
No I didn't. I used another theme as basis for this one and the vagrant file was there since the beginning.
